src/desktop/apps/artwork/components/artists/index.jade

```diff
@@ -2,32 +2,35 @@ include ../../../../components/side_tabs/mixins
 
 if artwork.artists.length
   for artist in artwork.artists
-    - var tabs = helpers.artists.build(artist)
+    if user && user.hasLabFeature('New Buy Now Flow')
+      != stitch.components.ArtworkArtistInfo({ artistID: artist.id })
```

Comment:
Does this have to be gated behind this feature?

Reply:
Yah